tests/large/test_ide.py

```diff
@@ -31,26 +31,25 @@
 logger = logging.getLogger(__name__)
 
 
-class EclipseIDETests(LargeFrameworkTests):
-    """The Eclipse distribution from the IDE collection."""
-
+class BaseLargeIdeTest(LargeFrameworkTests):
     TIMEOUT_INSTALL_PROGRESS = 120
     TIMEOUT_START = 60
     TIMEOUT_STOP = 60
 
     def setUp(self):
         super().setUp()
-        self.installed_path = os.path.expanduser("~/tools/ide/eclipse")
-        self.desktop_filename = "eclipse.desktop"
+        self.name = ""
+        self.command_name = ""
+        self.installed_path = ""
+        self.desktop_filename = ""
```

Comment:
To answer your comment: even if it can be seen as not as optimal to duplicate some infos here, keep in mind that those tests are large tests. They are independant of any information that umake is providing (they just set in stone some user experience). So, we want to avoid loading anything computed by the umake modules and really fake user's typing, without using any of their properties.

Also note that the runner can be on another machine that the destination, and this one could have some frameworks that are not loaded in memory as requirements are not available.